dev/az/datarepo-api.yaml

```diff
@@ -0,0 +1,35 @@
+---
+image:
+  tag: 1.220.0
```

Comment:
If you remove these two lines then the version will automatically bump according to the chart version.